src/enums/README.md

```diff
@@ -37,7 +37,12 @@ True unless the field is [specified with `SField::notSigning`](https://github.co
 
 ## Transaction Results
 
-TODO
+See:
+
+- https://github.com/ripple/rippled/blob/develop/src/ripple/protocol/TER.h
+- https://xrpl.org/transaction-results.html
+
+TODO: Write a script to read rippled's source file and generate the necessary mapping.
 
 ## Transaction Types
 
```

Comment:
I see this is still TODO, but maybe start with just a link to https://github.com/ripple/rippled/blob/develop/src/ripple/protocol/TxFormats.h